pkg/vendir/fetch/githubrelease/sync.go

```diff
@@ -165,17 +210,70 @@ func (d Sync) matchesAssetName(name string) (bool, error) {
 	return false, nil
 }
 
+func (d Sync) fetchTagSelection() (string, error) {
+	opt := &github.RepositoryListByOrgOptions{
```

Comment:
who should be using this? ListTags?

Reply:
can we modify tagSelection test to pick up tag from non-0th page?

Reply:
Whoops yeah, should be passed to list tags. 

I *think* it was , will need to revisit and also make it more explicit though, especially if we'll increase the page size used